demos/text_detection_demo/cpp_gapi/main.cpp

```diff
@@ -0,0 +1,349 @@
+        const char kPadSymbol = FLAGS_pad[0];
+        std::string trSymbolsSet;
+        if (!tryReadVocabFile(FLAGS_m_tr_ss, trSymbolsSet)) {
+            trSymbolsSet = FLAGS_m_tr_ss;
+        }
+        if (trSymbolsSet.find(kPadSymbol) != trSymbolsSet.npos) {
+            throw std::invalid_argument("Symbols set for the Text Recongition model must not "
+                                        "contain the reserved symbol " + std::string{ kPadSymbol });
+        }
+
+        custom::NetsConfig config(FLAGS_m_td, FLAGS_m_tr);
+        if (tdEnabled) {
+            config.getTDinfo();
+            config.configureTD(FLAGS_d_td, FLAGS_w_td, FLAGS_h_td);
+            slog::info << "The Text Detection model " << FLAGS_m_td << " is loaded to " <<
+                FLAGS_d_td << slog::endl;
+        }
+        if (trEnabled) {
+            config.getTRinputInfo();
+            if (trComposite) {
+                config.getTRcompositeInfo(encoderOutputNames, decoderInputNames,
+                                          decoderOutputNames, FLAGS_tr_pt_first, kPadSymbol,
+                                          trSymbolsSet, FLAGS_dt);
+                config.configureTRcomposite(FLAGS_d_tr);
+                slog::info << "The Composite Text Recognition Encoder model " << FLAGS_m_tr
+                    << " is loaded to " << FLAGS_d_tr << slog::endl;
+                slog::info << "The Composite Text Recognition Decoder model "
+                    << config.decoderModelPath << " is loaded to " << FLAGS_d_tr << slog::endl;
+            } else {
+                config.getTRoutputInfo(FLAGS_tr_o_blb_nm, FLAGS_tr_pt_first, kPadSymbol,
+                                       FLAGS_start_index, trSymbolsSet);
+                config.configureTR(FLAGS_d_tr);
+            slog::info << "The Text Recognition model " << FLAGS_m_tr << " is loaded to " <<
+                FLAGS_d_tr << slog::endl;
+            }
+        }
+
+        std::vector<cv::RotatedRect> emptyBox =
+            { {cv::Point2f(0.0f, 0.0f), cv::Size2f(0.0f, 0.0f), 0.0f} };
+        /** ---------------- Main graph of demo ---------------- **/
+        // Graph input
+        cv::GMat in;
+        // Graph outputs
+        auto outs = gapiStreaming ? cv::GOut(cv::gapi::copy(in)) : cv::GOut();
+
+        // Size of frame
+        cv::GOpaque<cv::Size> size = cv::gapi::streaming::size(in);
+        cv::GArray<cv::RotatedRect> rrs;
+        if (tdEnabled) {
+            // Text detection
+            cv::GMat inDet, det1, det2;
+            if (1 == config.tdInputChannels) {
+                inDet = cv::gapi::BGR2Gray(in);
+            } else {
+                inDet = in;
+            }
+            std::tie(det1, det2) = cv::gapi::infer<nets::TextDetection>(inDet);
+            // ROI for each text piece
+            rrs = custom::DetectionPostProcess::on(det1, det2, size, config.tdInputSize,
+                                                   segmConfThreshold, linkConfThreshold,
```

Comment:
```suggestion
                                                   static_cast<float>(FLAGS_cls_pixel_thr),
                                                   static_cast<float>(FLAGS_link_pixel_thr),
```
So you can remove `segmConfThreshold` and `linkConfThreshold`